tests/models/Document.test.js

```diff
@@ -89,6 +89,26 @@ const test = async app => {
     await tap.equal(url, documentObject.url)
   })
 
+  await tap.test('Delete all documents of a publication', async () => {
+    const numDeleted = await Document.deleteDocumentsByPubId(
+      urlToId(publication.id)
+    )
+
+    const docDeleted = await Document.byId(documentId)
+
+    await tap.equal(numDeleted, 1)
+    await tap.ok(docDeleted.deleted)
+  })
```

Comment:
can you also add a test where a publication has more than one document? Just to be sure.